services/ui-src/src/utils/forms/forms.ts

```diff
@@ -10,7 +10,26 @@ import {
   TextAreaField,
 } from "components";
 // types
-import { AnyObject, FormField } from "types";
+import { AnyObject, FieldChoice, FormField } from "types";
+
+const initializeChoiceFieldControl = (fields: FormField[]) => {
```

Comment:
this method initialized choice fields to be controlled inputs (necessary for our hydration and state handlers)